OWML.Common/OWML.Common.csproj

```diff
@@ -68,6 +72,42 @@
     <Compile Include="Menus\IModTextInput.cs" />
     <Compile Include="Menus\IModToggleInput.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="RegistrationCode.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Assembly-CSharp, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\OW.Unity.Dlls.1.0.5\lib\net35\Assembly-CSharp.dll</HintPath>
+    </Reference>
+    <Reference Include="OW.Unity.Dlls, Version=1.0.5.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\OW.Unity.Dlls.1.0.5\lib\net35\OW.Unity.Dlls.dll</HintPath>
+    </Reference>
+    <Reference Include="UnityEngine, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\OW.Unity.Dlls.1.0.5\lib\net35\UnityEngine.dll</HintPath>
+    </Reference>
+    <Reference Include="UnityEngine.AudioModule, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\OW.Unity.Dlls.1.0.5\lib\net35\UnityEngine.AudioModule.dll</HintPath>
+    </Reference>
+    <Reference Include="UnityEngine.CoreModule, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\OW.Unity.Dlls.1.0.5\lib\net35\UnityEngine.CoreModule.dll</HintPath>
+    </Reference>
+    <Reference Include="UnityEngine.IMGUIModule, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\OW.Unity.Dlls.1.0.5\lib\net35\UnityEngine.IMGUIModule.dll</HintPath>
+    </Reference>
+    <Reference Include="UnityEngine.PhysicsModule, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\OW.Unity.Dlls.1.0.5\lib\net35\UnityEngine.PhysicsModule.dll</HintPath>
+    </Reference>
+    <Reference Include="UnityEngine.TextRenderingModule, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\OW.Unity.Dlls.1.0.5\lib\net35\UnityEngine.TextRenderingModule.dll</HintPath>
+    </Reference>
+    <Reference Include="UnityEngine.UI, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\OW.Unity.Dlls.1.0.5\lib\net35\UnityEngine.UI.dll</HintPath>
+    </Reference>
+    <Reference Include="UnityEngine.UIModule, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\OW.Unity.Dlls.1.0.5\lib\net35\UnityEngine.UIModule.dll</HintPath>
+    </Reference>
+    <Reference Include="UnityEngine.UnityWebRequestWWWModule, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\OW.Unity.Dlls.1.0.5\lib\net35\UnityEngine.UnityWebRequestWWWModule.dll</HintPath>
+    </Reference>
```

Comment:
The common project shouldn't need these references?

Reply:
pretty sure it did that automatically

Reply:
also, it's the same in your branch, sooooooooo you are a hypocrite :P

Reply:
GitHub gets extremely weirded out by this section for some reason 🤷

Reply:
Huh, you're right. Weird that it's green 🤷